data_classes_test.py

```diff
@@ -0,0 +1,78 @@
+"""
+Unit tests for data classes
+
+Implemented:
+    Data
+
+TODO:
+    DataController
+    Dataset
+    Portfolio
+"""
+
+import unittest
+from datetime import datetime
+from thousandaire.data_classes import Data
+
+class TestData(unittest.TestCase):
+    """
+    Unit test object for Data.
+    """
+    def setUp(self):
+        self.data = Data('test', ['buy', 'sell'])
+        self.data_list = [
+            [(datetime(2020, 1, 27), 27, 27), (datetime(2020, 1, 28), 28, 28)],
+            [(datetime(2020, 1, 29), 29, 29), (datetime(2020, 1, 30), 30, 30)]]
+
+    def equal_judgment(self, response, answer):
```

Comment:
Because property of this method is different for 'test_method', I separate it from the 'test_method' part. Thus, it is not placed in dictionary order (In the order, it should be placed after test_method).

Reply:
Either ```append``` or ```extend``` will use same judgment more than one time, I extract it to a method.

Reply:
Can you name it `assertEqaulContent`?